search/indexing_api_test.py

```diff
@@ -215,38 +216,43 @@ def test_add_edx_record(self):
         Test that cached edX records are indexed after being added
         """
         program_enrollment = ProgramEnrollmentFactory.create()
-        for edx_cached_model_factory in [CachedCertificateFactory, CachedEnrollmentFactory]:
+        for edx_cached_model_factory in [CachedCertificateFactory, CachedEnrollmentFactory, CachedCurrentGradeFactory]:
             assert es.search()['total'] == 1
             course = CourseFactory.create(program=program_enrollment.program)
             course_run = CourseRunFactory.create(course=course)
             edx_cached_model_factory.create(user=program_enrollment.user, course_run=course_run)
+            index_program_enrolled_users([program_enrollment])
```

Comment:
Do we want to create the timestamp too? I'm not sure if it's relevant for these tests